R/find.match.R

```diff
@@ -1,38 +1,38 @@
+find_min_position <- function(distances) {
+    position <- which(distances == min(distances))[1]
```

Comment:
use which.min

Reply:
resolved in [4fd428f](https://github.com/RECETOX/recetox-aplcms/pull/193/commits/4fd428f79d85f15e2760d2755843dff298e9ec98)